pkg/operator/certrotation/signer.go

```diff
@@ -95,15 +96,25 @@ func (c RotatedSigningCASecret) EnsureSigningCertKeyPair(ctx context.Context) (*
 
 		LabelAsManagedSecret(signingCertKeyPairSecret, CertificateTypeSigner)
 
-		modified = true
+		updateRequired = true
 		signerUpdated = true
```

Comment:
do we really need this var ?

I think it could be replaced by `creationRequired || updateRequired`, right ?

Reply:
Yes. `updateRequired` means "any part of the secret has changed", but `signerUpdated` is specific to certificate part (we need to pass it on to trigger CA bundle / target cert updates)

Reply:
sure, I think that the following code:
```
	// at this point, the secret has the correct signer, so we should read that signer to be able to sign
	signingCertKeyPair, err := crypto.GetCAFromBytes(signingCertKeyPairSecret.Data["tls.crt"], signingCertKeyPairSecret.Data["tls.key"])
	if err != nil {
		return nil, signerUpdated, err
	}

	return signingCertKeyPair, signerUpdated, nil
```

could be replaced be:
```
	// at this point, the secret has the correct signer, so we should read that signer to be able to sign
	signingCertKeyPair, err := crypto.GetCAFromBytes(signingCertKeyPairSecret.Data["tls.crt"], signingCertKeyPairSecret.Data["tls.key"])
	if err != nil {
		return nil, creationRequired || updateRequired, err
	}

	return signingCertKeyPair, creationRequired || updateRequired, nil
```

and we could remove `signerUpdated`, am I right ?

Reply:
ok, do you also want to move the refactor commit to a new PR ?

Reply:
Lets leave this version as is and experiment if `signerUpdated` can be removed in a separate PR.

I think we want more specific messages for signer/cabundle/target too, i.e. `Updated contents` and `Updated metadata` probably